src/sagemaker/pytorch/estimator.py

```diff
@@ -298,6 +299,11 @@ def __init__(
             )
         self.compiler_config = compiler_config
 
+        if "profiler_config" in kwargs:
```

Comment:
Can we add this directly to base estimator class in https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/estimator.py, so that we can avoid framework specific repition now and in future?

Reply:
Will give it  a shot.

Reply:
Actually, I need a framework_version, which is a field which is not present in Abstract Framework.